pkg/kubectl/cmd/util/openapi/openapi_test.go

```diff
@@ -34,9 +34,10 @@ var _ = Describe("Reading apps/v1beta1/Deployment from openAPIData", func() {
 		Expect(err).To(BeNil())
 		instance, err = openapi.NewOpenAPIData(s)
 		Expect(err).To(BeNil())
+		fmt.Fprintf(GinkgoWriter, fmt.Sprintf("CHAO: instance.GroupVersionKindToName=%#v\n", instance.GroupVersionKindToName))
 	})
 
-	deploymentName := "io.k8s.kubernetes.pkg.apis.apps.v1beta1.Deployment"
+	deploymentName := "io.k8s.api.apps.v1beta1.Deployment"
```

Comment:
@mbohlool With the external API moved to k8s.io/api, the names in openapi change. Is there any backward compatibility concern?